runtime/gc_realtime/Scheduler.cpp

```diff
@@ -107,7 +110,7 @@ MM_Scheduler::getParameter(uintptr_t which, char *keyBuffer, I_32 keyBufferSize,
 		{
 			omrstr_printf(keyBuffer, keyBufferSize, "Scheduling Method");
 			I_32 len = (I_32)omrstr_printf(valueBuffer, valueBufferSize, "TIME_BASED with ");
```

Comment:
please convert various I/U_32/64 types to u/int32/64_t
seems like UDATA has been converted already